ngraph/core/include/ngraph/pattern/op/skip.hpp

```diff
@@ -45,6 +45,20 @@ namespace ngraph
                     set_output_type(0, arg.get_element_type(), arg.get_partial_shape());
                 }
 
+                Skip(const OutputVector& args, ValuePredicate pred)
```

Comment:
Why do we need this constructors in this review?

Reply:
it's required to make label_on_skip test pass. Since v1/v3 Broadcast has 3 input nodes, it cannot be matched with the Skip pattern we've had so far (which takes only 1 node as an input `Skip(const Output<Node>& arg...`)